submissions/danieljnj/Week1/Fizzbuzz.java

```diff
@@ -0,0 +1,22 @@
+
+public class FizzBuzzMain {
+	
+public static void main (String [] args) {
+	
+	for ( int fb = 1; fb <= 100; fb++) {          
+		
+	if (((fb % 3) == 0) && ((fb % 5) == 0)) {   
+		System.out.println ("FizzBuzz");
+	}
+		else if ((fb % 3) == 0){
```

Comment:
Indentation seems off here, can you please confirm and fix. The if and else statements should be at the same indentation level.